node/src/components/storage/config.rs

```diff
@@ -15,7 +15,7 @@ const APPLICATION: &str = "casper-node";
 
 const DEFAULT_MAX_BLOCK_STORE_SIZE: usize = 483_183_820_800; // 450 GiB
 const DEFAULT_MAX_DEPLOY_STORE_SIZE: usize = 322_122_547_200; // 300 GiB
-const DEFAULT_MAX_BLOCK_HEIGHT_STORE_SIZE: usize = 10_485_100; // 10 MiB
+const DEFAULT_MAX_BLOCK_HEIGHT_STORE_SIZE: usize = 10_485_760; // 10.48576 MiB
```

Comment:
I like the attention to detail, but... it'd be 10.48576 MB, not MiB.  `10_485_760` actually _is_ exactly 10 MiB (10 * 2^20).  I don't know where I got `10_485_100` from originally! :grimacing: